App/QmlImports/easyAnalysis/App/Elements/ChartViewAnalysis.qml

```diff
@@ -129,10 +129,38 @@ ColumnLayout {
                 anchors.margins: -extraPadding
                 anchors.bottomMargin: showDiff ? -4*extraPadding : -extraPadding
                 antialiasing: true // conflicts with useOpenGL: true in ScatterSeries
-                legend.visible: false
                 backgroundRoundness: 0
                 backgroundColor: "transparent"
                 titleFont: commonFont
+                legend.visible: false
+
+                // Custom Legend
+                Item {
+                    y: topChart.plotArea.top + 10
+                    x: topChart.plotArea.right - width - 10
+                    width: childrenRect.width
+                    height: childrenRect.height
+
+                    // Background
+                    Rectangle {
+                        width: parent.width
+                        height: parent.height
+                        opacity: 0.75
+                        color: "white"
+                        border.color: "#ddd"
+                    }
+
+                    // Info
+                    Grid {
+                        padding: 10
+                        spacing: 10
+                        columns: 2
+                        Text { text: "\u25A0"; color: obsArea.color; font.pointSize: 15 }
+                        Text { text: obsArea.name; font: commonFont}
+                        Text { text: "\u25A0"; color: calcSeries.color; font.pointSize: 15 }
+                        Text { text: calcSeries.name; font: commonFont}
```

Comment:
One can also set font.pointSize via the commonFont property as font.pointSize: commonFont.pointSize